src/automaton.jl

```diff
@@ -24,13 +24,20 @@ function get_ntrans(autom::AutomatonList)
     return length(autom.transitions)
 end
 
+# In add_trans and add_translist:
 # Do not check that source, symbol, target are "inbounds"
 # Assumes not add twice same transition...
-function add_transition!(autom::AutomatonList, source, symbol, target)
+function add_trans!(autom::AutomatonList, source, symbol, target)
```

Comment:
Add some point, we might want to merge the interface with the one in HybridSystems so we'd rather stay close:
https://blegat.github.io/HybridSystems.jl/latest/lib/methods/#HybridSystems.add_transition!

Reply:
changed